ddtrace/tracer/textmap.go

```diff
@@ -261,9 +268,47 @@ func (p *propagator) injectTextMap(spanCtx ddtrace.SpanContext, writer TextMapWr
 	for k, v := range ctx.baggage {
 		writer.Set(p.cfg.BaggagePrefix+k, v)
 	}
+	if p.cfg.MaxTagsHeaderLen <= 0 {
+		return nil
+	}
+	// propagate trace tags
+	s := p.marshalPropagatingTags(ctx)
+	if len(s) > 0 {
```

Comment:
```suggestion
	if s := p.marshalPropagatingTags(ctx); len(s) > 0 {
```
The `if` improvement is just a nit. Feel free to ignore. The comment on the other hand can be removed. It's clearly visible from the code that `s` gets written.